README.md

```diff
@@ -4,36 +4,64 @@ description: Learn more at baseline-protocol.org
+Version 1.0 of the Baseline Protocol has been released. It is composed of a set of 6 core packages that are open-sourced, under the [CC0 1.0 Universal public domain](https://github.com/ethereum-oasis-op/baseline/blob/main/LICENSE) dedication.&#x20;
 
 **You can find more about the** **source code** [**here**](docs/baseline-protocol-code/packages/)**.**
 
 ## The Baseline Protocol Standard
 
-The [Baseline Protocol Specifications ](https://github.com/eea-oasis/baseline-standard)were completed in September 2021 and is undergoing ratification to become an [OASIS Open Standard](https://www.oasis-open.org/standards/), available under the [CC0 1.0 Universal public domain](https://github.com/eea-oasis/baseline/blob/main/LICENSE) dedication.
+The [Baseline Protocol Specifications ](https://github.com/ethereum-oasis-op/baseline-standard) were completed in September 2021 and is undergoing ratification to become an [OASIS Open Standard](https://www.oasis-open.org/standards/), available under the [CC0 1.0 Universal public domain](https://github.com/ethereum-oasis-op/baseline/blob/main/LICENSE) dedication.
 
-The Baseline Protocol Standard contains three specifications - [CORE](https://github.com/eea-oasis/baseline-standard/tree/main/core), [API](https://github.com/eea-oasis/baseline-standard/tree/main/api), and [CCSM](https://github.com/eea-oasis/baseline-standard/tree/main/ccsm), that provide the requirements of a compliant Baseline Protocol Implementation (BPI).
+The Baseline Protocol Standard contains three specifications - [CORE](https://github.com/ethereum-oasis-op/baseline-standard/tree/main/core), [API](https://github.com/ethereum-oasis-op/baseline-standard/tree/main/api), and [CCSM](https://github.com/ethereum-oasis-op/baseline-standard/tree/main/ccsm), that provide the requirements of a compliant Baseline Protocol Implementation (BPI).
 
-**Find full details on the** [**Baseline Protocol Standard here**](https://github.com/eea-oasis/baseline-standard)**.**
+**Find full details on the** [**Baseline Protocol Standard here**](https://github.com/ethereum-oasis-op/baseline-standard)**.**
 
 ## The Baseline Protocol Reference Implementations
 
-A growing number of Baseline Protocol Reference Implementations can be found [here](docs/bri/overview-of-reference-implementations.md). Demos and examples can be found [here](https://github.com/ethereum-oasis/baseline/tree/master/examples). These are intended to help you understand baselining and give you ideas for your implementation needs.
+A growing number of Baseline Protocol Reference Implementations can be found [here](docs/bri/overview-of-reference-implementations.md). Demos and examples can be found [here](https://github.com/ethereum-oasis-op/baseline/tree/master/examples). These are intended to help you understand baselining and give you ideas for your implementation needs.
```

Comment:
Inlining links to text. It's mostly a stylistic convention, but also an accessibility consideration that is real.

Perhaps not the single most important thing to do for accessibility, it represents a real improvement in experience: screen-reader users are accustomed to go through a list of links instead of the whole page, and "here, here, here here" isn't useful enough, so forces them back to processing the entire content. 

```suggestion
A growing number of [Baseline Protocol Reference Implementations](docs/bri/overview-of-reference-implementations.md), along with [demonstrations and examples]https://github.com/ethereum-oasis-op/baseline/tree/master/examples) are available to help you understand baselining and give you ideas for your implementation needs.
```